### ISSUE TITLE
Add Accessibility Cheatsheet

### ISSUE THREAD

**Issue by logeshpaul** [logeshpaul/Frontend-Cheat-Sheets#3]
Accessibility is always important and this should be taken care in all web applications.

Link - http://bitsofco.de/2015/the-accessibility-cheatsheet/

**Comment by LizaVP** [comment]
Hello, I think I received a few emails from you in error.

Have a great day,
Liza

**Comment by logeshpaul** [comment]
Hey @LizaVP Might be you are watching this repository so you are getting notification for the new changes being pushed to the repository.

If you don't want to receive updates go to this url - https://github.com/logeshpaul/Frontend-Cheat-Sheets and click on 'Not Watching' button. You won't receive any updates!

Have a great day too!

**Comment by LizaVP** [comment]
Doh, my bad. You're right! Sorry. I don't use Git much.

Thank you,
Liza

On Wednesday, June 17, 2015, Logesh Paul notifications@github.com wrote:

> Hey @LizaVP https://github.com/LizaVP Might be if you watching this
> repository so you are getting notification for the new changes being pushed
> to the repository.
> 
> If you don't want to receive updates go to this url -
> https://github.com/logeshpaul/Frontend-Cheat-Sheets and click on 'Not
> Watching' button. You won't receive any updates!
> 
> Have a great day too!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/logeshpaul/Frontend-Cheat-Sheets/issues/3#issuecomment-112878430
> .

## 

Liza V. Pantschenko
Graphic Designer
CT & NYC
lvpdesign@gmail.com

Sent from a mobile device!
